GitUI/Strings.cs

```diff
@@ -32,6 +32,7 @@ internal sealed class Strings : Translate
         private readonly TranslationString _local = new TranslationString("Local");
         private readonly TranslationString _tag = new TranslationString("Tag");
         private readonly TranslationString _remote = new TranslationString("Remote");
+        private readonly TranslationString _contScrollToNextFileOnlyWithAlt = new TranslationString("Continuous scroll to next file only with ALT button");
```

Comment:
Maybe?
```suggestion
        private readonly TranslationString _contScrollToNextFileOnlyWithAlt = new TranslationString("Enable continuous scroll by holding ALT button");
```

Reply:
The functionality do not require ALT by default (but you could of course use the key), no key in addition to the mousewheel.
This option is to only use the scrolling with ALT and I believe the suggestion is misleading (but current text is not refined

Important to use something reasonable, functionality can be adapted to what is easiest to define here...

Reply:
In this case I propose we reword to
```suggestion
        private readonly TranslationString _contScrollToNextFileOnlyWithAlt = new TranslationString("Enable continuous scroll without ALT button");
```
and invert the setting. I wouldn't like the automatic continuous scroll enabled by default, it is confusing.